samples/snippets/csharp/System.Reflection.Metadata/SequencePoint/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;

```

Comment:
Can you put the snippet and project file in a subfolder of "SequencePoint" called "Overview"? Just in case we add other snippets for members of SequencePoint.